src/app/[locale]/_component/selectIdp/idpList.tsx

```diff
@@ -21,7 +25,21 @@ type SpidLevelL3 = {
 export type SpidLevels = SpidLevelL1 | SpidLevelL2 | SpidLevelL3;
 
 export function IdpList({ spidLevel }: IdpList) {
+  const { userLogged } = useLogin();
+  const token = storageTokenOps.read();
+  const taxCode = token ? userFromJwtToken(storageTokenOps.read()).taxCode : undefined;
+
+  const savePrivilegesData = () => {
+    if (taxCode) {
+      storagePrivilegeOps.write({
+        previous: userLogged?.spidLevel || undefined,
+        identity: MD5(taxCode).toString(),
+      });
+    }
+  };
+
   const getSPID = (IDP: IdentityProvider) => {
+    savePrivilegesData();
```

Comment:
I see when we save this date, but I cannot see where we remove this information after a successful login. Will this be the subject of another PR?

Reply:
I've just seen that this is handled here: https://github.com/pagopa/io-web-profile/pull/47 🙂